src/views/AddRecipe.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+    <div class="addRecipe">
+        <b-container fluid v-if="!loggedUserId">
+            <br />
+            <b-row style="padding-bot: 30px;">
+                <b-col cols="12">
+                    <h1>{{ $t('addRecipe.login') }}</h1>
+                </b-col>
+            </b-row>
+            <b-row>
+                <b-col cols="12">
+                    <router-link :to="'myAccount'">
+                        <b-button variant="danger">{{ $t('account.login') }}</b-button>
+                    </router-link>
+                </b-col>
+            </b-row>
+            <hr />
+        </b-container>
+        <b-container fluid v-else>
+            <b-row id="pagedesc">
+                <h1>{{ $t('addRecipe.pagedesc') }}</h1>
+            </b-row>
+            <b-form @submit="recipeSubmit">
+                <b-row>
+                    <b-col cols="12" lg="6">
+                        <b-form-group id="recipeNameGroup"
+                                        v-bind:label="$t('addRecipe.name')"
+                                        label-for="recipeName">
+                            <b-form-input id="recipeName"
+                                            type="text"
+                                            v-model="name"
+                                            required>
+                            </b-form-input>
+                        </b-form-group>
+                    </b-col>
+                    <b-col cols="12" lg="6">
+                        <b-form-group id="recipeTypeGroup"
+                                        v-bind:label="$t('addRecipe.type')"
+                                        label-for="recipeType">
+                            <b-form-select id="recipeType"
+                                        required
+                                        v-model="type">
+                                        <option v-for="cat in categories" v-bind:key="cat.id" :value="cat.id">{{$t(cat.i18n_name)}}</option>
+                            </b-form-select>
+                        </b-form-group>
+                        <br />
+                        <p class="text-danger"> {{ error }} </p>
+                    </b-col>
+                    <b-col cols="12" lg="6">
+                        <b-form-group v-bind:label="$t('addRecipe.duration')"
+                                        label-for="duration">
+                            <b-form-input id="duration"
+                                        v-model="duration"
+                                        type="text"
```

Comment:
Text => number? Ako nemaš neku drugu verifikaciju...
Možda i neki limit na tipa 600 minuta, čisto da ne bi bilo moguće staviti bolesne stvari...

Reply:
e hteo sam da stavim neki time input al to stavi pm i am tkd ne može, nem pojma kako to da stavim pa sam samo tako